scene/gui/slider.cpp

```diff
@@ -411,6 +451,12 @@ void Slider::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "scrollable"), "set_scrollable", "is_scrollable");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "tick_count", PROPERTY_HINT_RANGE, "0,4096,1"), "set_ticks", "get_ticks");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "ticks_on_borders"), "set_ticks_on_borders", "get_ticks_on_borders");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "ticks_position", PROPERTY_HINT_ENUM, "Bottom,Top,Both,Center"), "set_ticks_position", "get_ticks_position");
```

Comment:
You didn't update the hint with correct names.
Although, since the hint is pretty much dynamic, you could improve it and use `_validate_property()` to display different values for HSlider and VSlider.